packages/manager/modules/enterprise-cloud-database/src/card/card.component.js

```diff
@@ -0,0 +1,21 @@
+import controller from './card.controller';
+import template from './card.html';
+
+export default {
+  bindings: {
+    id: '@?',
+    name: '@?',
+    label: '@',
+    image: '@?',
+    disabled: '<?',
```

Comment:
Could you please order them alphabetically ? It would be easier to look into it :)